src/components/pages/DashboardAlerts/AlertsDummyData.js

```diff
@@ -0,0 +1,90 @@
+export const dummyData = [
+  {
+    clubid: 1,
```

Comment:
Maybe add a comment to the code to understand what the dummyData represents.

Reply:
I'll add that right now!